clang/lib/ExtractAPI/DeclarationFragments.cpp

```diff
@@ -1621,6 +1621,9 @@ DeclarationFragmentsBuilder::getSubHeading(const NamedDecl *Decl) {
              cast<CXXMethodDecl>(Decl)->isOverloadedOperator()) {
     Fragments.append(Decl->getNameAsString(),
                      DeclarationFragments::FragmentKind::Identifier);
+  } else if (dyn_cast<TagDecl>(Decl) &&
```

Comment:
Could this use `isa<TagDecl>` instead? The extraneous pointer cast is probably not a big deal in the broad scheme, but using `dyn_cast` as a boolean check just feels weird to me.

Reply:
absolutely